# ex. use with the default collision check benchmark
gnuplot_config:
  plots:
    - filters:
        - ns: query/request
          val: All (Contact Only)
          predicate: "=="
      labels:
        - query/scene
      legends:
        - query/collision_detector
      metrics:
        - type: boxplot
          name: time
          title: '{/:Bold Collision (Self + Environment)}\nTime (sec) for request \"All (Contact Only)\" against scenes and collision detectors'
        - type: boxplot
          name: contact_count
          title: '{/:Bold Collision (Self + Environment)}\nCountact count for request \"All (Contact Only)\" against scenes and collision detectors'
  options:
    terminal: QT
    n_row: 2
    n_col: 1
    size:
      x: 1280
      y: 720
